packages/beta/src/__tests__/cogniteClient.int.spec.ts

```diff
@@ -3,10 +3,12 @@
 import CogniteClient from '../cogniteClient';
 import { setupLoggedInClient } from './testUtils';
 
+const BASE_URL = 'https://greenfield.cognitedata.com';
```

Comment:
Could we define this at only one place? DRY principle.

Preferably a env var

Reply:
Sure